quick_sort/QuickSort.java

```diff
@@ -1,9 +1,38 @@
 public class QuickSort {
 
+    private static int compare(int[] arr, int i, int j) {
```

Comment:
Is `compare` a good name?
Or should it be `greaterIndex` or something relatable to what it is returning?